crypto/evp/pmeth_fn.c

```diff
@@ -153,6 +154,17 @@ static void *evp_signature_from_dispatch(int name_id,
                 break;
             signature->dupctx = OSSL_get_OP_signature_dupctx(fns);
             break;
+        case OSSL_FUNC_SIGNATURE_GET_PARAMS:
+            if (signature->get_params != NULL)
+                break;
+            signature->get_params = OSSL_get_OP_signature_get_params(fns);
+            break;
+        case OSSL_FUNC_SIGNATURE_GETTABLE_PARAMS:
+            if (signature->gettable_params != NULL)
+                break;
+            signature->gettable_params
+                = OSSL_get_OP_signature_gettable_params(fns);
+            break;
```

Comment:
We have a check below that if "get_ctx_params" is present then so must "gettable_ctx_params" and vice versa. It seems we should have something similar here. Similarly for the other dispatch functions.

Reply:
Fixed